### ISSUE TITLE
Clickatell vs Nexmo in SA

### ISSUE THREAD

**Issue by devdurbs** [etiennemarais/ama#2]
Hi Etienne

I noticed you built a wrapper for Clickatell for Laravel. I was wondering why you felt it was necessary over and above the built-in Nexmo?

The reason I ask, did you feel Clickatell was a better bet for use in SA, or is it much of the same between the two providers?

Just curious
Thanks
Ray

**Comment by etiennemarais** [comment]
Hi @devdurbs 

Great question. On comparing the two providers I haven't really done so to be honest. At least writing this for the notifications project, I haven't brought into consideration pricing and those types of things at all.

My reason were a lot less analytical than you'd might expect 😄 I did an integration ~3 years ago that was exactly this and I wanted to contribute to an open source project mainly by learning how a contribution would look like at scale and what tools they use.

I had the code lying around and I got the idea to spend a Friday evening hacking at it to get it working.

I felt that giving people a choice would be more suited to the type of open source project this was. I also learnt what type of open source tools are available like StyleCI, Scrutinizer, Travis, how they worked and what the github integrations looked like. That was a pretty cool experience to deal with international developers and see how they respond to questions and the way they handled their communities.

On the topic of comparing them? I always do that analysis on my own before every project I pick up. Some projects might not have the budget to send sms @ 34c a pop and some might. You are going to have to define that on your own with your team taking the specific project considerations into consideration.

E.
